addon/mixins/route.js

```diff
@@ -199,6 +200,17 @@ export default Ember.Mixin.create({
   },
 
   /**
+   Call additional functions after finding the infinityModel in the Ember data store.
+
+   @method afterInfinityModel
+   @param {Function} infinityModelPromise The resolved result of the Ember store find method. Passed in automatically.
+   @return {Ember.RSVP.Promise}
+   */
+  afterInfinityModel(infinityModelPromiseResult) {
+    return infinityModelPromiseResult;
```

Comment:
regardless of the ember-infinity implementation (always flowing through the hook vs only flowing through it if defined) the developer must always be sure the return implements `get('content')` though... right?

The difference is if we *always* flow through the hook, the developer can call `_super` (conditionally, perhaps) if they feel like it.

```
afterInfinityModel(newObjects) {
  if (mercuryInRetrograde) {
    return doWork(newObjects)
  } else {
    this._super.apply(...arguments);
  }
}
```

If imagine always flowing through a function is also `sliiiiiiiiightly` quicker for the machine